### PR TITLE
add biosses templates

### PR DESCRIPTION
**Pull request by bwang482**
Creating templates for biosses (biomedical subgroup)

### CODE REVIEW

**Comment by bwang482** [comment]
Thanks for the feedback @stephenbach !

I thought the UUIDs were unique per dataset. I have changed the dataset name and UUIDs now.

**Comment by stephenbach** [comment]
Thanks. Just noticing some more things. This must have been copied from something before 0.1. There shouldn't be any answer_choices_key fields in the YAML either. It would also be nice if you updated the choices in prompt field as well. And give the templates meaningful names.

File: promptsource/templates/biosses/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: false
+    name: similarity binary
+    reference: stsb_multi_mt
+  f2b20779-4ac9-41d9-9660-b9c5223fe9c1: !Template
+    answer_choices: null
+    id: f2b20779-4ac9-41d9-9660-b9c5223fe9c1
+    jinja: 'Rate the similarity of these two sentences: ({{"1.0"}} being the lowest
```
Comment (stephenbach) [inline promptsource/templates/biosses/templates.yaml:141]:
```suggestion
    jinja: 'Rate the similarity of these two sentences: ({{"0.0"}} being the lowest
```

File: promptsource/templates/biosses/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      original_task: true
+    name: same thing scoring
+    reference: stsb template from FLAN
+  5c53ce9b-45f6-41ab-9da7-9c24f0f6f56d: !Template
+    answer_choices: no ||| yes
+    id: 5c53ce9b-45f6-41ab-9da7-9c24f0f6f56d
+    jinja: "(1) {{sentence1}} \n(2) {{sentence2}}\n\nDo these two sentences convey\
+      \ the same information? |||\n\n{{answer_choices[0 if score < 2.5 else 1]}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
```
Comment (stephenbach) [inline promptsource/templates/biosses/templates.yaml:89]:
```suggestion
      choices_in_prompt: false
```

File: promptsource/templates/biosses/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      original_task: true
+    name: rate with question first
+    reference: stsb_multi_mt
+  fc22748c-72c0-4727-bc4e-53aae4449bef: !Template
+    answer_choices: no ||| yes
+    id: fc22748c-72c0-4727-bc4e-53aae4449bef
+    jinja: Do you think "{{sentence1}}" and "{{sentence2}}" express the same thing?
+      ||| {{answer_choices[0 if score < 2.5 else 1]}}
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
```
Comment (stephenbach) [inline promptsource/templates/biosses/templates.yaml:159]:
```suggestion
      choices_in_prompt: false
```

File: promptsource/templates/biosses/templates.yaml [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      original_task: false
+    name: same meaning binary
+    reference: paws_wiki from FLAN
+  e22d8c63-3184-40df-84c2-6800960496a7: !Template
+    answer_choices: no ||| yes
+    id: e22d8c63-3184-40df-84c2-6800960496a7
+    jinja: Do "{{sentence1}}" and "{{sentence2}}" seem similar to you ? ||| {{answer_choices[0
+      if score < 2.5 else 1]}}
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
```
Comment (stephenbach) [inline promptsource/templates/biosses/templates.yaml:132]:
```suggestion
      choices_in_prompt: false
```

**Review by stephenbach — APPROVED** [review]
Awesome, thanks for working so hard on this! Just fixed a few remaining things. Merging now!

**Pull request merged**